app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -43,187 +43,10 @@ def self.any_without_iso_datastores?
     without_iso_datastores.count > 0
   end
 
-  def supports_port?
-    true
-  end
-
-  def supported_auth_types
-    %w(default metrics)
-  end
-
   def supports_authentication?(authtype)
```

Comment:
It's not part of this PR, but this method will clash once there is a feature called `authentication`. Because the SupportsFeatureMixin will add a method `supports_authentication?` :worried: 

Maybe you can rename that in another PR?